tests/Unit/TracerTest.php

```diff
@@ -195,4 +195,17 @@ public function testUnfinishedSpansCanBeFinishedOnFlush()
         $this->assertSame('root', $sent[0][0]->getOperationName());
         $this->assertSame('child', $sent[0][1]->getOperationName());
     }
+
+    public function testSpanStartedAtRootCanBeAccessedLater()
+    {
+        $tracer = new Tracer(new NoopTransport());
+        $span = $tracer->startRootSpan(self::OPERATION_NAME);
```

Comment:
If you approve the suggestions above, then this should be `$scope`. I think that  we should refactor that  part returning the span that let you access the scope, but this is very in the future